vendor.mod

```diff
@@ -56,6 +56,7 @@ require (
 	github.com/moby/swarmkit/v2 v2.0.0-20220721174824-48dd89375d0a
 	github.com/moby/sys/mount v0.3.3
 	github.com/moby/sys/mountinfo v0.6.2
+	github.com/moby/sys/sequential v0.0.0-20220829095930-b22ba8a69b30
```

Comment:
No release yet, but I'll do a release once review of all PR's is looking good, and will update to `v0.5.0` (once release) in a follow-up.